fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -433,22 +428,49 @@ def convertArmatureToGeolayout(armatureObj, obj, convertTransformMatrix, camera,
             None,
             None,
             None,
-            meshGeolayout.nodes[i],
+            i,
             [],
             name,
             meshGeolayout,
             geolayoutGraph,
             infoDict,
             convertTextureData,
         )
+
+    def walk(node, fMeshes):
```

Comment:
you should add type hints here (if possible, if it's too annoying then it's fine), this applies to the functions that are added in this pr